tutorials/inverse/20_dipole_fit.py

```diff
@@ -57,8 +59,31 @@
 mri_pos = mne.head_to_mri(dip.pos, mri_head_t=trans,
                           subject=subject, subjects_dir=subjects_dir)
 
+# In the meantime let's find an anatomical label for the best fitted dipole
+best_dip_id = dip.gof.argmax()
+best_dip_mri_pos = mri_pos[best_dip_id]
+
+# Load parcellation:
+atlas_fname = op.join(subjects_dir, subject, 'mri', 'aparc.a2009s+aseg.mgz')
+parcellation_img = nibabel.load(atlas_fname)
+
+# Load freesurface atlas LUT
+lut_inv_dict = read_freesurfer_lut()[0]
+label_lut = {v: k for k, v in lut_inv_dict.items()}
+
+# Find voxel for dipole position
+mri_vox_t = np.linalg.inv(parcellation_img.header.get_vox2ras_tkr())
+vox_dip_pos_f = mne.transforms.apply_trans(mri_vox_t,best_dip_mri_pos)
+vox_dip_pos = vox_dip_pos_f.astype(int)
+
+# Get voxel value and label from LUT
+vol_values = parcellation_img.get_fdata()[tuple(vox_dip_pos.T)]
+label = label_lut.get(vol_values, 'Unknown')
```

Comment:
I find this code a bit scary for average users and this usecase seems quite common.

how about we add a

`find_label_in_annot(pos, subject='sample', annot="a2009s+aseg")`

function?

wdyt?

Reply:
Yes it is a good idea, I'm just a little busy with my projects and wanted to start with a small contribution which would be helpful.
More like `find_label_in_annot(pos, subject='sample', annot="a2009s+aseg", subject_dir=dir)`

But where to put such function? In what package? What test case to add and where? Just to test a few known positions?